scripts/ipet-parse

```diff
@@ -120,52 +173,26 @@ if __name__ == '__main__':
     if type(arguments.logfiles) == io.TextIOWrapper and not arguments.debug:
         logger.setLevel(logging.ERROR)
 
-    if arguments.logfiles is None:
-        logger.info("No testruns specified, exiting")
-        sys.exit(0)
 
     # initialize an experiment
-    experiment = Experiment(validatedual = arguments.validatedual, gaptol = arguments.gaptol)
+    experiment = setup_experiment(arguments, verbose=True)
```

Comment:
Note that I use a verbose initialization of an experiment here. I prefer to keep it here so that there is some information that readers and a solu file are included at the beginning of the parsing process. Other than that, this is actually a redudant experiment initialization, after all, if not for input from standard input.